test/e2e/framework.go

```diff
@@ -2892,6 +2892,14 @@ func (data *TestData) checkAntreaAgentInfo(interval time.Duration, timeout time.
 			// keep trying
 			return false, nil
 		}
+		//validate that the podRef in AntreaAgentInfo matches the name of the current Pod for the Node
```

Comment:
nit: insert a space after `//` and capitalize the first letter to make the comment more formal.